INSTALL.md

```diff
@@ -30,37 +28,52 @@ make
 ./porymap.app/Contents/MacOS/porymap
 ```
 
-## Windows
-
+## Windows 7+
 Install [Qt development tools](https://www.qt.io/download-qt-installer), and use Qt Creator, the official Qt IDE, for development purposes.
 
-## Ubuntu
 
-You need to install Qt. The minimum supported version is currently Qt 5.14.2. You can check your Qt version
-with `qtdiag` or `qmake --version`.
+## Gentoo Linux
+You need to install Qt dev tools. You can set version of your Qt packages with flags such as `qt6`.
+25.04.2 is porymap's minumum supported dev-util/kdevelop version.
+https://wiki.gentoo.org/wiki/Qt
 
 ```bash
-sudo apt-get install qt6-declarative-dev
-# if your distro does not have qt6-declarative-dev, try sudo apt-get install qtdeclarative5-dev
+sudo emerge dev-util/kdevelop
+sudo emerge --depclean
+qmake6
+make
+./porymap
+```
 
-git clone https://github.com/huderlem/porymap
-cd porymap
```

Comment:
Please include these steps. They're often assumed, but we get many users for whom this process is unfamiliar (and the steps were added recently because people had missed them).